test/boost/cql_query_test.cc

```diff
@@ -1921,7 +1921,9 @@ SEASTAR_TEST_CASE(test_select_multiple_ranges) {
 SEASTAR_TEST_CASE(test_validate_keyspace) {
     return do_with_cql_env([] (cql_test_env& e) {
         return make_ready_future<>().then([&e] {
-            return e.execute_cql("create keyspace kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkssssssssssssssssssssssssssssssssssssssssssssss with replication = { 'class' : 'NetworkTopologyStrategy', 'replication_factor' : 1 };");
+            constexpr auto NAME_MAX_LENGTH = 192; // check schema::NAME_LENGTH
```

Comment:
Please just use schema::NAME_LENGTH instead of this constant - this is one of the few benefits of C++ unit tests, they can use things from the C++ code.